admin/src/components/drawer/index.js

```diff
@@ -237,13 +254,32 @@ export default (props) => {
         {user.role === ROLES.HEAD_CENTER && headCenter({ user, onClick: handleClick })}
         {user.role === ROLES.SUPERVISOR && supervisor({ user, onClick: handleClick })}
         {user.role === ROLES.RESPONSIBLE && responsible({ user, onClick: handleClick })}
-        {user.role === ROLES.ADMIN && admin({ onClick: handleClick })}
-        {[ROLES.REFERENT_DEPARTMENT, ROLES.REFERENT_REGION].includes(user.role) && referent({ onClick: handleClick })}
+        {user.role === ROLES.ADMIN && admin({ onClick: handleClick, newTickets, openedTickets })}
+        {[ROLES.REFERENT_DEPARTMENT, ROLES.REFERENT_REGION].includes(user.role) && referent({ onClick: handleClick, newTickets, openedTickets })}
       </ul>
     </Sidebar>
   );
 };
 
+const mapDispatchToProps = (dispatch) => ({
+  dispatchTickets: (tickets) => {
+    const newTickets = tickets?.filter((ticket) => ticket.state_id === 1);
+    const openedTickets = tickets?.filter((ticket) => ticket.state_id === 2);
+    dispatch({
+      type: 'FETCH_TICKETS',
+      payload: {
+        tickets,
+        new: newTickets.length,
+        open: openedTickets.length,
```

Comment:
si `tickets` est null, ca crash. Car il ne fait pas de filter, donc newTickets est null, et on ne peut pas prend `.length` de `null` :(

je ne sais pas si ca peut arriver, mais dans le doute je mettrais des `?.`

```suggestion
        new: newTickets?.length,
        open: openedTickets?.length,
```